### PR TITLE
chore: Fix ts

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [x] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

TS update which break internal ts definition. This fix dose not affect export interface.

### CODE REVIEW

File: components/affix/index.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ class Affix extends React.Component<AffixProps, AffixState> {
 
   fixedNode: HTMLDivElement;
 
-  private timeout: number;
+  private timeout: any;
```
Comment (afc163) [inline components/affix/index.tsx:63]:
```
NodeJS.Timeout
```

Comment (zombieJ) [reply components/affix/index.tsx:63]: 它的确是 `number` 类型，我不想勉强……

**Comment by afc163** [comment]
要写 changelog：https://github.com/ant-design/ant-design/issues/29487

File: components/affix/index.tsx (hunk 1 above)
Comment (hengkx) [reply components/affix/index.tsx:63]: 这个是好烦人

**Pull request merged**